addons/l10n_lu/account.account.template-2011.csv

```diff
@@ -898,7 +898,7 @@ lu_2011_account_6555,6555,Escomptes et frais sur effets,lu_2011_account_655,othe
 lu_2011_account_6556,6556,Escomptes accordés,lu_2011_account_655,other,account_type_2011_6_charge_finance,FALSE,"account_financial_report_135,account_financial_report_abr_135"
 lu_2011_account_6558,6558,Intérêts sur autres emprunts et dettes,lu_2011_account_655,other,account_type_2011_6_charge_finance,FALSE,"account_financial_report_135,account_financial_report_abr_135"
 lu_2011_account_656,656,Pertes de change,lu_2011_account_65,other,account_type_2011_6_charge_finance,FALSE,"account_financial_report_135,account_financial_report_abr_135"
-lu_2011_account_657,657,Quote-part de perte dans les entreprises collectives (autres que les sociétés de capitaux),lu_2011_account_65,other,account_type_2011_6_charge_finance,FALSE,"account_financial_report_135,account_financial_report_abr_135"
+lu_2011_account_657,657,Quote-part de perte dans les entreprises collectives (autres que les sociétés de capitaux),lu_2011_account_65,other,account_type_2011_6_charge_finance,FALSE,"account_financial_report_135,account_financial_report_abr_135,account_financial_report_164,account_financial_report_abr_164"
```

Comment:
@laetitia-gangloff I'd say you need to remove `account_financial_report_135,account_financial_report_abr_135` here since this account now goes in `account_financial_report_164,account_financial_report_abr_164`